swift-paperless/Views/TLSListView.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  TLSListView.swift
+//  swift-paperless
+//
+//  Created by Nils Witt on 24.06.24.
+//
+
+import os
+import SwiftUI
+
+struct TLSIdentity: Identifiable, Equatable {
+    var name: String
+    var identity: SecIdentity
+
+    var id: String { name }
+}
+
+enum CertificateState {
+    case notloaded
+    case wrongPassword
+    case valid
+}
+
+struct TLSListView: View {
+    @State var idenities: [TLSIdentity] = []
```

Comment:
```suggestion
    @State var identities: [TLSIdentity] = []
```

Requires some changes down in the file.